test/functions/iterators/bradc/captureEmpty.chpl

```diff
@@ -1,4 +1,4 @@
-iter foo() {
+iter foo(): nothing {
   writeln("In foo");
 }
 
```

Comment:
It would be fun to extend this test with an int-yielding iterator, ex:
```chpl
iter bar(): int { writeln("In bar"); }
var B = bar();
writeln((B, B.domain, B.eltType:string));
```

However since this would not be executed within this test, I suggest making this addition to another test below, instead of bar() reusing the iterator already in that test, `i11e()`: test/functions/vass/proc-iter/error-no-yield-in-iter-1.chpl